tests/phpunit/tests/interactivity-api/wpInteractivityAPIFunctions.php

```diff
@@ -332,16 +332,18 @@ public function test_process_directives_when_block_is_filtered() {
 				),
 			)
 		);
-		function test_render_block_data( $parsed_block ) {
+
+		$test_render_block_data = function ( $parsed_block ) {
```

Comment:
@jrfnl what about making this `static`?

```suggestion
		$test_render_block_data = static function ( $parsed_block ) {
```

Reply:
Good point! Thanks for catching that! Fixed now.